lib/embedded_ansible.rb

```diff
@@ -11,13 +17,114 @@ def self.enabled?
   end
 
   def self.running?
-    # TODO
-    false
+    services.map { |service| LinuxAdmin::Service.new(service).running? }.inject(:&)
```

Comment:
Yeah, not sure why I did it that way ... `all?` is probably the better way to go ...